pages/getting-started/writing-your-first-test.md

```diff
@@ -0,0 +1,92 @@
+# Escrevendo o primeiro teste
+```
+O que você vai aprender
+
+- Como começar a testar um novo projeto com Cypress
+- Como são reportados testes com sucesso e falhas
+- Testando a navegação web, buscando por elementos no DOM e escrevendo asserções
+```
+
+[Vídeo de exemplo.](https://vimeo.com/237115455)
```

Comment:
hmmm I don't know.. maybe it could be helpful as least as a visual resource. I think it helps even if the person just watch the actions in the videos and don't listen to it. Seeing things happening can be more helpful than words sometimes :) And by the way there are videos without audio, like the other one in this page. What do you think?

Reply:
@lucastfa-toptal should be a good ideia, probably we'll need to verify if we are missing any videos link to existing docs also